wallet/chain/p/wallet.go

```diff
@@ -507,7 +503,7 @@ func (w *wallet) IssueTx(
 	}
 
 	if txStatus.Status != status.Committed {
-		return errNotCommitted
+		return fmt.Errorf("not committed: %s", txStatus.Reason)
```

Comment:
Instead of getting rid of the `errNotCommited` error type we could export it and wrap this error so the caller can handle this case if needed

Reply:
I meant that the caller could do something like this if we exported the error:
```
err := w.IssueTx()
if errors.Is(err, ErrNotCommited) {
...
}
```